CHANGELOG.md

```diff
@@ -23,6 +23,7 @@
 
 ### BUG FIXES
 - [state] [\#458](https://github.com/line/ostracon/pull/458) Fix the thread-unsafe of PeerState logging
+- [indexer] [#498](https://github.com/tendermint/tendermint/pull/498) Fix overriding tx index of duplicated txs
```

Comment:
Unnecessary since here is for CHANGELOG of v1.0.7 and this will be written on making release tags.